core/crypto/vrf/p256/p256.go

```diff
@@ -74,7 +74,8 @@ func GenerateKey() (vrf.PrivateKey, vrf.PublicKey) {
 		return nil, nil
 	}
 
-	return &PrivateKey{key}, &PublicKey{&key.PublicKey}
+	return &PrivateKey{PrivateKey: key},
+		&PublicKey{PublicKey: &key.PublicKey}
```

Comment:
These probably don't need to be on separate lines.  Even with the field names added, the return statement isn't long enough to warrant a line split.

Reply:
done